packages/e2e/features/ui/components/authenticator/sign-in-totp-mfa.feature

```diff
@@ -63,6 +63,7 @@ Feature: Sign In with TOTP MFA
     And I type a new "email"
     And I type my password
     And I confirm my password
+    Given I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.SignUp" } }' with fixture "sign-up-with-email"
```

Comment:
There might be a few other places we are doing this without mocking. But this is a good start and as long as the tests still pass 👍